Classes/MDMPersistenceController/MDMPersistenceController.m

```diff
@@ -85,19 +109,19 @@ - (NSPersistentStoreCoordinator *)setupNewPersistentStoreCoordinator {
                                              NSMigratePersistentStoresAutomaticallyOption:@YES
                                              };
     NSError *persistentStoreError;
-    NSPersistentStore *persistentStore = [persistentStoreCoordinator addPersistentStoreWithType:NSSQLiteStoreType
+    NSPersistentStore *persistentStore = [persistentStoreCoordinator addPersistentStoreWithType:storeType
                                                                                   configuration:nil
                                                                                             URL:self.storeURL
                                                                                         options:persistentStoreOptions
                                                                                           error:&persistentStoreError];
-    if (persistentStore == nil) {
+    if (persistentStore == nil && [storeType isEqualToString:NSSQLiteStoreType]) {
```

Comment:
The reason was the below code was only concerned with SQLite type .. but I see what you mean about `persistentStore` being `nil`.
There's a "some really bad has happened" logic inside this condition .. I'll see if it makes sense to pull that out to cover both SQLite and InMemoryStore types.